dts/arm/st/g0/stm32g0.dtsi

```diff
@@ -308,6 +308,31 @@
 			label = "ADC_1";
 			#io-channel-cells = <1>;
 		};
+
+		dma1: dma@40020000 {
+			compatible = "st,stm32-dma-v2";
+			#dma-cells = <4>;
+			reg = <0x40020000 0x400>;
+			interrupts = <9 0 10 0 10 0 11 0 11 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x1>;
+			dma-requests = <5>;
```

Comment:
Would you suggest to not include it in stm32g0.dtsi, but instead leave a note that it differs between socs, such that people know this can change once they look into stm32g0?
All Socs but g030, and g031 have 7 channels. Would you prefer setting it only in g03x, and g05x or also in g07x and g0bx?

Reply:
Could ST guys weight in on this. I believe there is currently not one single established way of how it is handled.
It's just a trivia, but I think it could be beneficial if it would be done consistently for properties across all series?

a) if a property is known to change for different socs of a serie. Should a default value already be set in stm32XY.dtsi(stm32g4.dtsi, stm32h7.dtsi,...)?
Or should this file only have a remark that it changes across different socs of the serie?
b) Should a property that changes across different socs be overwritten in each stm32XYabc.dtsi file or only when strictly necessary(the value is different from the value set in the include file)?